Sample JSON Rules/if-switchLevel-remains-lessThan.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "If switch level is less than 25% for 20 seconds in d1, turn off d2",
+    "actions": [
+      {
+        "if": {
+            "remains":{
+                "duration":{
+                    "value": {"integer":20},
+                    "unit": "Second"
+                },
+                "lessThan":{
+                    "left": {
+                        "device": {
+                          "devices": [
+                            "{{ device-id }}"
```

Comment:
For consistency, I believe this should be:

```
"{{ d1 }}"
```

or

```
"{{ device1 }}"
```

but, if you choose the last option, then Rule name should be updated accordingly.